wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -1301,6 +1306,62 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:complexType>
 			</xs:element>
 			<!--===============================-->
+
+			<xs:element name="GetMulticastAudioDecoderConfigurations">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="ConfigurationToken" type="tt:ReferenceToken"  minOccurs="0">
+							<xs:annotation>
+								<xs:documentation>Unique token of the multicast audio decoder configuration. If not passed, device shall respond with all configurations.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>	
+			<xs:element name="GetMulticastAudioDecoderConfigurationsResponse">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="Configurations" type="tt:MulticastAudioDecoderConfiguration" minOccurs="0" maxOccurs="unbounded" >
+							<xs:annotation>
+								<xs:documentation>This message contains the list of multicast audio decoder configurations.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+			
+			<xs:element name="GetMulticastAudioDecoderConfigurationOptions">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="ConfigurationToken" type="tt:ReferenceToken"  minOccurs="0">
+							<xs:annotation>
+								<xs:documentation>This response contains the available MulticastAudioDecoderConfigurationOptions. If a multicast audio decoder configuration is specified, the options shall concern that particular configuration.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="GetMulticastAudioDecoderConfigurationOptionsResponse">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="Options" type="tt:MulticastAudioDecoderConfigurationOptions" maxOccurs="unbounded"/>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+
+			<xs:element name="SetMulticastAudioDecoderConfiguration">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="Configuration" type="tt:MulticastAudioDecoderConfiguration">
```

Comment:
Shouldn't this be plural Configurations and have maxOccurs="unbounded" to match the Get operation?

Reply:
In most of the set operation, its for one configuration, example "SetAudioDecoderConfiguration" or "SetVideoEncoderConfiguration", followed the same.

Reply:
If the Set operation only sets set (add or modify) one configuration and not all of them I think we are missing a RemoveMulticastAudioDecoderConfiguration operation.

Reply:
@johado , 
Following our last f2f meeting, we've decided to streamline this feature with just three methods:

- GetMulticastAudioDecoderConfigurations - Lists all pre-supported configurations, similar to videoencoderconfigurations.
- SetMulticastAudioDecoderConfiguration
- GetMulticastAudioDecoderConfigurationOptions

and will not be supporting the Add and Remove methods since each configuration can be enabled or disabled, making the Remove method unnecessary.

If you still feel Add/Remove methods are necessary, we can discuss in our VEWG telco.